doc/CMakeLists.txt

```
@@ -330,23 +363,39 @@ add_custom_target(
         ${SPHINXOPTS_EXTRA}
         ${KCONFIG_RST_OUT}
         ${HTML_DIR}/kconfig
+  DEPENDS kconfig-content
   USES_TERMINAL
 )
-add_dependencies(kconfig-html kconfig-content)
+
+add_custom_target(
+  kconfig-html-all
+  DEPENDS
+    kconfig-html
+)
 
 #-------------------------------------------------------------------------------
 # Global targets
 
+add_dependencies(zephyr-html-all kconfig-html-all)
+add_dependencies(mcuboot-html-all kconfig-html-all)
+add_dependencies(nrfxlib-inventory-all kconfig-html-all)
+add_dependencies(nrf-html-all nrfxlib-inventory-all kconfig-html-all)
+add_dependencies(nrfxlib-html-all nrf-html-all)
+
 add_custom_target(
   build-all
```

Should we add `ALL` here, so that users may simply run `ninja` / `make` and get the complete doc set built.

Currently, users have to run `ninja build-all`
```suggestion
  build-all ALL
```

---

changed, thanks!